components/coding/profileV2/AssignmentSectionComponent.tsx

```diff
@@ -16,6 +16,30 @@ import {
 import { useAuth } from "../../../lib/authContext";
 import ExpandableContainer from "../ExpandableContainer";
 
+export const returnWrapStyling = (
+  assignment: UserAssignmentSubmissionsData
+) => {
+  let wrapStyle = "";
+  if (assignment.coding_assignment.assignment_name.length >= 3) {
+    wrapStyle = "truncate";
+  } else {
```

Comment:
no need for this else clause; if the if statement returns false, it will return an empty string wrapStyle

Reply:
great catch.  removed!